src/Screens/App/Screens/BooksDashboard/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
```

Comment:
Split this component using Smart & Dumb pattern.
For reference: https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0